bindings/node/native/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "node-reconfix"
+version = "0.1.0"
+authors = ["Aaron Brodersen <aaron@resin.io>"]
+license = "MIT"
+build = "build.rs"
+
+[lib]
+name = "node_reconfix"
+crate-type = ["dylib"]
+
+[build-dependencies]
+neon-build = { git = "https://github.com/resin-io-modules/neon.git", branch = "fork" }
+
+[dependencies]
+neon = { git = "https://github.com/resin-io-modules/neon.git", branch = "fork" }
```

Comment:
Is it correct that both `neon-build` and `neon` refer to the same git repo?  (it probably is, I'm just checking this isn't a typo)

Reply:
@lurch Yeah, there are multiple crates in that git repo.